app/Http/Controllers/Api/V1/Integration/VideoDirectUrlController.php

```diff
@@ -99,8 +103,10 @@ private function getConfigObjectFromHtml($string, $start, $end)
     }
 
     /**
+     * Get Vimeo video object
```

Comment:
Add new line between title, description param and return.

Reply:
Done!